html/semantics/forms/form-submission-0/enctypes-helper.js

```diff
@@ -90,12 +126,23 @@
 
   // This function returns a function to add individual form tests corresponding
   // to some enctype.
-  // `expectedBuilder` is a function that takes two parameters: `expected` (the
-  // `expected` value of a test) and `serialized` (the actual form body
-  // submitted by the browser), and returns the correct form body that should
-  // have been submitted. This is necessary in order to account for th
+  // `expectedBuilder` is an optional callback that takes two parameters:
+  // `expected` (the `expected` property passed to a test) and `serialized` (the
+  // actual form body submitted by the browser, isomorphic-decoded). It must
+  // return the correct form body that should have been submitted,
+  // isomorphic-encoded. This is necessary in order to account for the
   // multipart/form-data boundary.
-  window.formSubmissionTemplate = (enctype, expectedBuilder) => {
+  window.formSubmissionTemplate = (
+    enctype,
+    expectedBuilder = (expected) => expected
+  ) => {
+    // `name` and `value` are the form entry's name and value – `name` must be a
+    // string, `value` can be either a string or a `File` object.
+    // `expected` is the expected form body – usually a `string`, but it can be
+    // anything depending on the `expectedBuilder` passed to
+    // `formSubmissionTemplate`.
+    // `formEncoding` is the character encoding used for submitting the form.
+    // `description` will be used as part of the testharness test's description.
```

Comment:
I think this documentation needs to be before window.formSubmissionTemplate, since this is documenting the return value of that helper function which callers actually do use.

Reply:
I thought having the description of the returned function's parameters there might make the docs for `formSubmissionTemplate` a bit unreadable, but it doesn't actually seem to be that bad.